packs/campaign-effects/resonant-reflection-reflection-of-life.json

```diff
@@ -30,6 +30,35 @@
                 "mode": "add",
                 "path": "system.attributes.hp.recoveryMultiplier",
                 "value": 1
+            },
+            {
+                "itemType": "action",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:slug:resonant-reflection"
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.ResonantReflection.ReflectionOfLife.Description"
+                    }
+                ]
+            },
+            {
+                "itemType": "action",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "item:slug:resonant-reflection"
+                ],
+                "property": "traits",
+                "value": "healing"
+            },
+            {
+                "allowDuplicate": false,
+                "key": "GrantItem",
+                "uuid": "Compendium.pf2e.actionspf2e.Item.Activate Resonant Reflection"
```

Comment:
Need to add a predicate for not having the item already, otherwise people will get spammed with the "you already have this action" message.

We'll have to add a RollOption RE to the action itself to predicate on.